packages/e2e-tests/src/specs/editor/publishingFlow.js

```diff
@@ -25,8 +25,55 @@ import {
   withPlugin,
   publishStory,
   getEditedPostContent,
+  withUser,
 } from '@web-stories-wp/e2e-test-utils';
 
+describe('Contributor User', () => {
+  let stopRequestInterception;
+
+  beforeAll(async () => {
```

Comment:
🤔  since this is to verify proper messaging for contributors, should it live in the `prepublishChecklist` file instead since this is to verify the publishing flow?